### DISCUSSION TITLE
Continued support from the Cloudmos Team

### DISCUSSION THREAD

**Discussion by lechenghiskhan** [akash-network/community#472] [Governance Proposals]
## Background

In August of 2023, [Proposal 216](https://www.mintscan.io/akash/proposals/216) passed and the Community Pool was used to help fund the acquisition of Cloudmos to further develop and integrate the widely used deployment tool. All code has since been made open source. Overclock Labs subsidized ⅓ of the acquisition price and compensated the Cloudmos team monthly for 6 months following the acquisition to fund further development and integration.

The Cloudmos team has received $ USD-denominated payments over 6 Months from the [recipient wallet address](https://www.mintscan.io/akash/address/akash1v6c77xl46cfwcxcg30urygvpn8sv8rp6mwaluz). The final payment comes at the end of January. Due to price appreciation, the wallet still contains ~197k AKT and should still be sizable after the last payment is made.

## Continued Funding

The Cloudmos team has committed to working on Akash Network for at least another 12 months alongside the OCL core team. The core team proposes repurposing the remaining funds for paying the Cloudmos team over the next 12 months (starting February 2024 through January 2025).

The Akash Network roadmap, which includes substantial contributions from the Cloudmos team and continued development to the Cloudmos client can be seen [here](https://github.com/orgs/akash-network/projects/5). 

The OCL core team looks forward to working with the Cloudmos team through 2024 as the development of Akash Network continues to advance.

**Comment by Dimokus88** [comment]
Cloudmos is an amazing tool for accessing the Akash marketplace!  Continuing to fund it is the right decision.

**Comment by zJuuu** [comment]
I support the continuation of the development of cloudmos.
A few questions about the development cost for 12 Months.
- Will it also be $ USD-denominated payments? If yes how many USD in total?
- Will the funds vest over the next 12 months?
